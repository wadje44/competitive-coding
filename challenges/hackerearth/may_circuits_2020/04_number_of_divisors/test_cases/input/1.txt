4
10 3
10 2
10 5
1000000000 97