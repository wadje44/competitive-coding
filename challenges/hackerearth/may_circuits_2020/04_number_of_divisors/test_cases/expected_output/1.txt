41
36
43
494897959532893312